druid/src/text/layout.rs

```diff
@@ -280,6 +284,17 @@ impl<T: TextStorage> TextLayout<T> {
             .unwrap_or_else(|| Line::new(Point::ZERO, Point::ZERO))
     }
 
+    /// Returns the link at the mouse position.
+    pub fn link_for_mouse_pos(&self, pos: Point) -> Option<&Link> {
```

Comment:
```suggestion
    pub fn link_for_pos(&self, pos: Point) -> Option<&Link> {
```

It is not *strictly* necessary that the position originate from a mouse?